test/e2e/network/endpointslice.go

```diff
@@ -143,7 +143,7 @@ var _ = SIGDescribe("EndpointSlice", func() {
 		}
 
 		// Expect EndpointSlice resource to be deleted when Service is.
-		if err := wait.PollImmediate(2*time.Second, 12*time.Second, func() (bool, error) {
+		if err := wait.PollImmediate(2*time.Second, 18*time.Second, func() (bool, error) {
```

Comment:
this is a surprisingly specific time... why not wait.ForeverTestTimeout (30 seconds)?

Reply:
Somehow I think I'd thought the only reason it would take longer than this would be a legitimate failure, clearly that's not always the case. Also, I didn't know about `wait.ForeverTestTimeout` - I've updated all 4 instances of 12 second times to use that instead, thanks!